lib/utils/cli.go

```diff
@@ -294,6 +294,38 @@ func InitCLIParser(appName, appHelp string) (app *kingpin.Application) {
 	return app.UsageTemplate(defaultUsageTemplate)
 }
 
+// UpdateAppUsageTemplate updates usage template for kingpin application.
+func UpdateAppUsageTemplate(app *kingpin.Application, args []string) {
+	context, err := app.ParseContext(args)
+	if err != nil {
+		return
+	}
+
+	// Update command printf width if longer than default.
+	commandWidth := 0
+	var commands []*kingpin.CmdModel
+	if context.SelectedCommand != nil {
+		commands = context.SelectedCommand.Model().FlattenedCommands()
+	} else {
+		commands = app.Model().FlattenedCommands()
+	}
+
+	for _, command := range commands {
+		if !command.Hidden && len(command.FullCommand) > commandWidth {
+			commandWidth = len(command.FullCommand)
+		}
+	}
+
+	template := defaultUsageTemplate
+	if commandWidth > defaultCommandPrintfWidth {
+		template = strings.ReplaceAll(template,
```

Comment:
A brief comment explaining the replace would help. Maybe:

```suggestion
		// Replaces "%-12s" (the default width in the template) with the larger width.
		template = strings.ReplaceAll(template,
```